pkgs/by-name/dp/dprint/plugins/default.nix

```diff
@@ -12,28 +16,26 @@ let
       license ? lib.licenses.mit,
       maintainers ? [ lib.maintainers.phanirithvij ],
     }:
-    fetchurl {
-      inherit
-        hash
-        url
-        pname
-        version
-        ;
-      name = "${pname}-${version}.wasm";
+    stdenv.mkDerivation (finalAttrs: {
+      inherit pname version;
+      src = fetchurl { inherit url sha256; };
+      dontUnpack = true;
       meta = {
-        inherit
-          description
-          license
-          maintainers
-          ;
+        inherit description license maintainers;
       };
+      buildPhase = ''
+        mkdir -p $out
+        cp $src $out/plugin.wasm
+        ln -s $out/plugin.wasm $out/plugin.wasm@${sha256}
+      '';
```

Comment:
this creates two files
 - `/nix/store/...-{pname}-{version}/plugin.wasm`
 - `/nix/store/...-{pname}-{version}/plugin.wasm@<sha256sum of plugin>`

Reply:
> `plugin.wasm@<sha256sum of plugin>`

Is this required? As far as I know, the style of sha256 suffix only used for process plugins.